test/cpp/end2end/tls_key_export_test.cc

```diff
@@ -163,6 +163,8 @@ class TlsKeyLoggingEnd2EndTest : public ::testing::TestWithParam<TestScenario> {
           GRPC_SSL_REQUEST_AND_REQUIRE_CLIENT_CERTIFICATE_AND_VERIFY);
       server_creds_options.watch_identity_key_cert_pairs();
       server_creds_options.watch_root_certs();
+      server_creds_options.set_min_tls_version(grpc_tls_version::TLS1_2);
```

Comment:
This can be a good example of e2e test, but it is testing some other functionality(TLS key exporting).
Could you please write another test similar to this one but testing the TLS version setting behavior? You can also move any shared code into a common place, similar to `h2_tls_common.h`

Reply:
I looked at several tests in this directory, it seems all tests construct their fixture classes independently. I followed the `grpc_auth_...` test and created a new `tls_creds_options_test.cc`. More tests could be added but for the purpose of this PR I put the tls version tests only. Let me know what you think.